src/styles/calendar.scss

```diff
@@ -0,0 +1,55 @@
+
+@import "./variables.scss";
+
+.calendar {
+
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  max-width: $container-width;
+  padding: 30px;
+  gap: 1px;
```

Comment:
you can use variable for this, if will be used a couple of times and can be changed

Reply:
What exactly?)

Reply:
gap, if you want to make your max-width changing on changes of gaps